src/audio_common.c

```diff
@@ -305,6 +305,8 @@ static bool audio_config_updated(SDL_Event *evt, void *arg) {
 			break;
 	}
 
+	mute_audio_handler();
```

Comment:
This looks like it works, however, the switch above no longer does anything useful and can be removed entirely. The `mute_audio_handler` code can just be moved into this function as well.